easybuild/easyconfigs/p/Phantompeakqualtools/Phantompeakqualtools-1.2.2-foss-2021b.eb

```
@@ -0,0 +1,54 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'Tarball'
+
+name = 'Phantompeakqualtools'
+version = '1.2.2'
+
+homepage = "https://github.com/kundajelab/phantompeakqualtools"
+
+description = """ It computes informative enrichment and quality measures for ChIP-seq/DNase-seq/FAIRE-seq/MNase-seq data."""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+
+# Download manually from https://github.com/kundajelab/phantompeakqualtools/releases
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['b31263b64aefe97bdc4d7fae138f515a7d7a60cd05031d38dd89a10f9ee10cd1']
+
+dependencies = [
+    ('R', '4.2.0'),
+    ('SAMtools', '1.15.1'),
+    ('R-bundle-Bioconductor', '3.15', '-R-4.2.0'),
```

The CI test suite is failing/complaining, as we usually stick to the same versions of dependencies within a toolchain. For instance, for `foss/2021b` the default for `SAMtools` is `1.14`. So this can be fixed using the following versions:

```suggestion
    ('R', '4.1.2'),
    ('SAMtools', '1.14'),
    ('R-bundle-Bioconductor', '3.14', '-%(rver)s'),
```
If this does really require a slightly newer version of these dependencies, exceptions can be added to the test suite.

---

Ah, yes, you're absolutely right, that should have included `-R`, as `rver` only includes the version number itself.